### ISSUE TITLE
Mobile compatibility?

### ISSUE THREAD

**Issue by arianaa30** [5starkarma/face-smoothing#3]
Interesting work. Do you think if it can be used in Android for real-time camera frames as well? Any clues or hints?

**Comment by 5starkarma** [comment]
I would assume that it would be more efficient to have ML models handle beautification these days. @arianaa30

**Comment by arianaa30** [comment]
Yeah I think so. They should be generally slower but not sure what these face filter apps do that make it realtime and fast
